src/hs_output.h

```diff
@@ -18,7 +18,7 @@ typedef struct hs_output
   FILE* fh;
   char* path;
   pthread_mutex_t lock;
-  size_t id;
```

Comment:
my new patch doesn't change `hs_output.h` anymore, so I won't address this comment myself in this PR